app/pods/agendas/template.hbs

```diff
@@ -19,12 +40,176 @@
+    @buttonText={{t "clear-filter"}}
+    @clearAction={{this.clearFilter}}
+  />
+{{/if}}
+<div class="auk-scroll-wrapper">
+  <div class="auk-scroll-wrapper__body">
+    {{#if @model.length}}
+      <div class="auk-u-ml-4">
+        <table
+          data-test-route-agendas-overview-data-table
+          class="auk-table auk-table--agenda auk-table--sticky-header"
+        >
+          <thead>
+            <tr>
+              <Utils::ThSortable
+                class="auk-table__col--3 auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="created-for.planned-start"
+                @onChange={{this.sortTable}}
+                @label={{t "agenda"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="created-for.is-final"
+                @onChange={{this.sortTable}}
+                @label={{t "status"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="created-for.kind.label"
+                @onChange={{this.sortTable}}
+                @label={{t "kind"}}
+              />
+              <Utils::ThSortable
+                class="auk-table__header-title"
+                @currentSorting={{this.sort}}
+                @field="modified"
+                @onChange={{this.sortTable}}
+                @label={{t "latest-modified"}}
+              />
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#if this.isLoadingModel}}
+              <tr>
+                <td
+                  data-test-route-agendas-overview-loader
+                  colspan="5"
+                >
+                  {{t "loading"}}
+                </td>
+              </tr>
+            {{else}}
+              {{#each @model as |agenda|}}
+                <tr
+                  class="auk-u-text-align--center auk-table--interaction-cursor auk-table--clickable-rows"
+                  {{! template-lint-disable no-invalid-interactive }}
+                  {{on "click" (fn this.onClickRow agenda)}}
+                >
+                  <td data-test-route-agendas-overview-row-title>
+                    <span class="auk-o-flex auk-o-flex--vertical-center auk-o-flex--spaced">
+                      {{#if agenda.status.isDesignAgenda}}
+                        <strong>{{t "design-agenda"}}</strong>
+                        <Auk::Pill @skin="active-future">{{agenda.name}}</Auk::Pill>
+                      {{else}}
+                        <strong>{{t "agenda"}}</strong>
+                        <Auk::Pill>{{agenda.name}}</Auk::Pill>
+                      {{/if}}
+                      <span>–</span>
+                      <span>
+                        {{moment-format agenda.createdFor.plannedStart "DD.MM.YYYY"}}
```

Comment:
Date format should not longer be provided, so we fallback to the configured default (as soon as #1295 has been merged)

Reply:
set it to dashes just so it fits with the other column, will remove it once #1295 gets merged